mantidproject/mantid
# plotting of python fit function

Enable the python FunctionWrappers to plot their function.

**To test:**

Build the documentation as well as Mantid. Run plot in the various manners described in ``concepts/FitFunctiondInPython.html`` for 
a FitFunctionWrapper you have created as described in the same document.
Check that all this plot fulfils to requirements in
https://github.com/mantidproject/documents/blob/master/Design/Fit%20functions%20in%20python.md

Fixes #20005 . 

**Release Notes** 
to do

---

#### Reviewer ####

Please comment on the following ([full description](http://www.mantidproject.org/Individual_Ticket_Testing)):

##### Code Review #####


##### Functional Tests #####


Does everything look good? Mark the review as **Approve**. A member of `@mantidproject/gatekeepers` will take care of it.
